ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -1280,20 +1280,17 @@ private void LoadDefinedNames(Workbook workbook)
                 {
                     string text = definedName.Text;
 
-                    if (!(text.Equals("#REF") || text.EndsWith("#REF!")))
+                    var localSheetId = definedName.LocalSheetId;
```

Comment:
Do I understand correctly that we will now load named ranges that are invalid? Previously we ignored them?